.github/workflows/CI.yml

```diff
@@ -68,3 +68,14 @@ jobs:
         run: sudo apt install -y dotnet6
       - name: Run script to make sure all files end with EOL 
         run: dotnet fsi eofConvention.fsx
+      - name: Install prettier
+        run: npm install prettier@2.8.3
+      - name: Change file permissions
+        # We need this step so we can change the files using `npx prettier --write` in the next step.
+        # Otherwise we get permission denied error in the CI.
+        run: sudo chmod 777 -R .
+      - name: Run "prettier" to check the style of our TypeScript code
+        run: |
+          sudo npx prettier --quote-props=consistent --write './**/*.ts'
+          git config core.fileMode false
+          git diff --exit-code './**/*.ts'
```

Comment:
@realmarv let's revisit this conversation sorry; if we increase the use of prettier then I'm not sure git diff can receive many arguments; so, just in case (and because it's more readable and self-explanatory anyway), let's rather use `git restore` (with the package.json files as arg) before `git diff` command.

Reply:
Doesn't `git restore` command reset all files to the last commit state?

Reply:
Read my comment again, I said **git restore (with the package.json files as arg)**

Reply:
Done